### ISSUE TITLE
AttributeError: 'JpegImageFile' object has no attribute 'shape'

### ISSUE THREAD

**Issue by RomRoc** [martinzlocha/anchor-optimization#2]
I run code in Python 3.6, I haven't 3.7 available,
When I run `python optimize_anchors.py annotations.csv` I get the error:

```
Using TensorFlow backend.
Loading object dimensions.
Traceback (most recent call last):
  File "optimize_anchors.py", line 137, in <module>
    scale = compute_resize_scale(img.shape, min_side=args.image_min_side, max_side=args.image_max_side)
AttributeError: 'JpegImageFile' object has no attribute 'shape'
```

**Comment by martinzlocha** [comment]
Hi yes the issue is that Jpegs don't have a shape. Instead they use `img.size`.

So you can replace img.shape by `(img.size[0], img.size[1], 3)`. Test it out and if it works feel free to make .a PR.

**Comment by iibrahimli** [comment]
added #3 which fixes this small issue

**Comment by martinzlocha** [comment]
Merged #3 so it is fixed now.
